sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -633,6 +633,10 @@ ALTER TABLE `users_secure` ADD COLUMN `login_work_area` text;
 ALTER TABLE `onsite_messages` CHANGE `sender_id` `sender_id` VARCHAR(64) NULL COMMENT 'who sent id';
 #EndIf
 
+#IfNotColumnType onsite_documents full_document MEDIUMBLOB
+ALTER TABLE `onsite_documents` CHANGE `full_document` `full_document` MEDIUMBLOB NULL DEFAULT NULL;
```

Comment:
hi @sjpadgett ,
remove the `NULL DEFAULT NULL`

Reply:
Also should put this at the bottom of the script (we keep it chronological so we don't go mad if need to figure things out)